wire-ios-transport/Source/Public/BackendInfo.swift

```diff
@@ -68,5 +71,18 @@ public enum BackendInfo {
         let storedValue = storage.integer(forKey: key.rawValue)
         return APIVersion(rawValue: Int32(storedValue))
     }
+}
+
+// MARK: - Mock
+
```

Comment:
Consider putting this code in an `#IF DEBUG` to prevent it from being used for some reason in production